app/frontend/layouts/student_course/components/StudentCourse__Header.res

```diff
@@ -59,7 +59,7 @@ let imageWrapperClasses = coverImage =>
 let renderCourseSelector = (currentCourseId, courses, coverImage, additionalLinks) => {
   let currentCourse = ArrayUtils.unsafeFind(
     c => CourseInfo.id(c) == currentCourseId,
-    "Could not find current course with ID " ++ (currentCourseId ++ " in StudentCourse__Header"),
+    tr("current_course_not_found") ++ (currentCourseId ++ " in StudentCourse__Header"),
```

Comment:
`current_course_not_found` is used in 3 files; It will be good to move it into the shared group;